lib/sourceParams.js

```diff
@@ -33,7 +33,7 @@ module.exports = function(app) {
     disableLogger: params.disableLogger || pkg.rooseveltConfig.disableLogger || false,
     noMinify: params.noMinify || pkg.rooseveltConfig.noMinify || false,
     enableValidator: params.enableValidator || pkg.rooseveltConfig.enableValidator || false,
-    htmlValidator: params.htmlValidator || pkg.rooseveltConfig.htmlValidator || {port: '8888', format: 'text', suppressWarnings: false},
+    htmlValidator: params.htmlValidator || pkg.rooseveltConfig.htmlValidator || {port: '8888', validatorAlwaysOn: false, format: 'text', suppressWarnings: false},
```

Comment:
- change `validatorAlwaysOn` to `separateProcess`
- document this in the README
- add an ability to enable/disable this param via command line as well